contribute/CONTRIBUTING-TO-K8S-DEMO.md

```diff
@@ -6,27 +6,28 @@ Kubernetes YAML files for running application using OpenEBS Volumes are located
 
 Each application example should comprise of the following:
 - K8s YAML file(s) for starting the application and its associated components. The volumes should point to OpenEBS Storage Class. If the existing storage-classes don't suit the need, create a new storage class in: [openebs-storageclasses.yaml](../k8s/openebs-storageclasses.yaml) 
-- K8s YAML files for starting a client that access the application. This is optional, in case the Application itself provides a mechanism like in case of Jupyter, Wordpress etc., When demonstrating database like applications like Cassandra, Reddis, it is recommended to have a mechanism to test that application is launched.   
-- Instruction Guide, that will help launch and verify the application
+- K8s YAML files for starting a client that access the application. This is optional, in case the application itself provides a mechanism like in Jupyter, Wordpress etc. When demonstrating database-like applications such as Cassandra, Redis, and so on, it is recommended to have a mechanism to test that the application is launched. 
+- Instruction Guide, that will help launch and verify the application.
 
 At a very high level, the process to contribute and improve is pretty simple:
-- Submit an Issue describing your proposed change. Or pick an existing issue tagged as [applications](https://github.com/openebs/openebs/labels/application)
-- Create your development branch
-- Commit your changes
-- Submit your Pull Request
-- Submit an issue to update the [User Documentation](https://github.com/openebs/openebs/blob/master/documentation/source/install/install_usecases.rst)
+- Submit an Issue describing your proposed change. Or pick an existing issue tagged as [applications](https://github.com/openebs/openebs/labels/application).
+- Create your development branch.
+- Commit your changes.
+- Submit your Pull Request.
+- Submit an issue to update the User Documentation.
 
-*Note: You could also help by just adding new issues for applications that are currently missing in the examples or by raising issues on existing applications for further enhancements.*
+***Note:** You could also help by just adding new issues for applications that are currently missing in the examples or by raising issues on existing applications for further enhancements.*
 
-The following sections describe some guidelines that can come in handy with the above process. 
-*Followed by the guidelines, is a cheat sheet with frequently used git commands.*
+The following sections describe some guidelines that are helpful with the above process. 
 
-## Submit an Issue describing your proposed change.
+*Following the guidelines, here is a [cheatsheet](./git-cheatsheet.md) with frequently used git commands.*
```

Comment:
Remove *

Reply:
* is used for italics in markdown. If we need italics then they would need be used in pairs.